wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,101 @@
+<template lang='pug'>
+  div.container
+      div.container-register
+        img.logo(alt="Wolox logo" src="../assets/logo.png")
+        form(v-on:click='handleSignUp')
```

Comment:
This should be on submit and we use the shorthand `@` for the `v-on` and `:` for the `v-bind`